packages/playwright-core/src/utils/hostPlatform.ts

```diff
@@ -55,7 +55,7 @@ export const hostPlatform = ((): HostPlatform => {
     return macVersion as HostPlatform;
   }
   if (platform === 'linux') {
-    const archSuffix = os.arch() === 'arm64' ? '-arm64' : '';
+    const archSuffix = '-' + os.arch();
```

Comment:
Looking at https://nodejs.org/api/os.html#osarch, there are quite a few more architectures. We should return `<unknown>` when its not `x64` or `arm64`.

Reply:
This does make the error message worse:

```diff
- Error: ERROR: Playwright does not support chromium on debian12-arm
+ Error: ERROR: Playwright does not support chromium on <unknown>
```

It would be nice to give a clearer error message here, but it's also not great to pretend that "debian12-arm" is a real thing.